### ISSUE TITLE
Wrong logout URL in disqus user menu

### ISSUE THREAD

**Issue by Globulopolis** [tsi/inlineDisqussions#11]
I'm not sure this is problem with this plugin or something wrong or at the disqus side.
Can you explain this issue?

**Comment by tsi** [comment]
What do you get exactly? I don't experience it.
This is probably not related to this plugin, maybe a cookie or browser issue...

**Comment by Globulopolis** [comment]
I think that the logout url must be redirect on the same page.
Example:
http://mysite.com -> http://disqus.com/logout/?redirect=http://mysite.com -> http://mysite.com

But second url on my page w/o original url. Just a http://disqus.com/logout/?redirect=

**Comment by tsi** [comment]
On the demo page what do you get? I get it right - http://disqus.com/logout/?redirect=http%3A%2F%2Ftsi.github.io%2FinlineDisqussions%2F

**Comment by Globulopolis** [comment]
Yep. It's working. The problem was on the Disqus server. If html request w/o referrer when the 'logout url' is empty.
